src/ws/cockpit.service.in

```diff
@@ -7,7 +7,6 @@ After=cockpit-wsinstance-http.socket cockpit-wsinstance-http-redirect.socket coc
 
 [Service]
 RuntimeDirectory=cockpit/tls
-RuntimeDirectoryMode=0700
```

Comment:
Does this mean that everyone on the system can see all the certificates of anyone logged into the system? Certificates typically contain user information. Is this okay?

Reply:
> Why are we opening up permissions here

So that cockpit-ws can check if it has a current certificate, and thus should attempt `tls-cert` authentication to cockpit-session. If we would always blindly do that, we'd get a lot of confusing error messages in the journal (the commit message explains this).

> should the comment below be removed too?

No, that applies to the `RUNTIME_DIRECTORY` variable below.

Reply:
Wrt. visibility: non-admin IPA users can see the certificates of all other IPA users with `ipa user-find` and `ipa user-show`. And of course running cockpit sessions can be seen by anyone in `loginctl` (ideally also `who`, but that doesn't work yet -- we have a ticket for that).

If/when we implement the nonce proposal, we can tighten this again. But until then I don't think this leaks any private information.

Reply:
On the other hand, "least privilege".

Reply:
Yep, fully agreed. It's part of the follow-up TODO list in the description.